src/features/shared-link-settings-modal/messages.js

```diff
@@ -58,7 +58,7 @@ const messages = defineMessages({
         id: 'boxui.share.sharedLinkSettings.passwordTitle',
     },
     vanityNamePlaceholder: {
-        defaultMessage: 'Enter a custom path',
+        defaultMessage: 'Enter a path name (12 or more characters)',
```

Comment:
should this say 12 to 30 characters, since there is an actual limit?

Reply:
holding for confirmation on this text string. might not include the limits at all

Reply:
on second thought, there is nothing in the component to enforce this currently, so this string acts as a recommendation. I think we should consider a refactor where client-side validation can be built against props. Any server-side validation can hook to the error props